README.md

```diff
@@ -250,14 +250,14 @@ Creates a new git commit/tag in the process of publishing to npm.
 
 More specifically, this command will:
 
-1. Publish each module in `packages` that has been updated since the last version to npm with the [dist-tag](https://docs.npmjs.com/cli/dist-tag) `lerna-temp`.
+- First, publish each module in `packages` that has been updated since the last version to npm with the [dist-tag](https://docs.npmjs.com/cli/dist-tag) `lerna-temp`.
```

Comment:
This temporary dist-tag strategy was actually inverted recently, which means that by default we _do not_ publish to the `lerna-temp` dist-tag first. Looks like we forgot a part of the docs to update!

I think we could make this much easier to read by entirely removing the outer list, so all that remains is the inner numbered list, which you should probably un-indent so Github doesn't try to get too fancy with it.